ports/esp32/boards/GENERIC_UNICORE/board.md

```diff
@@ -0,0 +1,3 @@
+The following files are daily firmware for single-core ESP32-based boards without external SPIRAM.
```

Comment:
This file is just copied and is not correct.  You can just leave it out and the default will be provided on the download page.

Reply:
Actually, it specifies `daily firmware for single-core ESP32s`. I think the single-core clarification is necessary because the generic firmware doesn't work in single-core models.

Reply:
Ah, sorry, you are right.